### PR TITLE
Add Argmax WhisperKit Snippet and Docs URL 

### PR DESCRIPTION
**Pull request by ardaatahan**
This PR:

1.  Adds a new `whisperkit` snippet in [model-libraries-snippets.ts](https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries-snippets.ts).
2. Adds snippet and docsUrl to `whisperkit` in `MODEL_LIBRARIES_UI_ELEMENTS` in [model-libraries.ts](https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries-snippets.ts).

### CODE REVIEW

File: packages/tasks/src/model-libraries-snippets.ts
Comment (ZachNagengast) [inline packages/tasks/src/model-libraries-snippets.ts:724]:
```suggestion
# Download and run inference using whisper base model
whisperkit-cli transcribe --audio-path /path/to/audio.mp3

# Or use a your preferred model variant
whisperkit-cli transcribe --model "large-v3" --model-prefix "distil" --audio-path /path/to/audio.mp3`
```

File: packages/tasks/src/model-libraries-snippets.ts
Comment (ZachNagengast) [inline packages/tasks/src/model-libraries-snippets.ts:728]:
```suggestion
# Or use your preferred model variant
whisperkit-cli transcribe --model "large-v3" --model-prefix "distil" --audio-path /path/to/audio.mp3 --verbose`
];
```

File: packages/tasks/src/model-libraries-snippets.ts
Comment (ZachNagengast) [inline packages/tasks/src/model-libraries-snippets.ts:721]:
```suggestion
brew install whisperkit-cli

# View all available inference options
whisperkit-cli transcribe --help
```

**Review by Vaibhavs10 — APPROVED** [review]
Thanks for updating the Docs and the Snippet! LGTM.
Let's wait for atleast one more approval before we merge.

I'll kick off the CI in the meantime.

**Review by Vaibhavs10 — COMMENTED** [review]
You can ignore the other tests, but do run the linter to fix the lint issues.

**Review by Vaibhavs10 — COMMENTED** [review]
The lint check passes now. Let's get another approval before we merge. Thanks again for improving the docs.

Failing tests don't matter in the context of this PR. 🤗

**Comment by Vaibhavs10** [comment]
cc: @Wauplin or @osanseviero - can one of you approve/ review this pls?

**Review by Wauplin — APPROVED** [review]
Looks good! Thanks @ardaatahan @ZachNagengast @Vaibhavs10 :hugs:

**Comment by Vaibhavs10** [comment]
Thank you! 🤗

**Pull request merged**
